[ADD] rental: product page

meval1006 opened the pull request:
- Copied and revised the Rental Order information from the Rental home page and made a new Products page under Rental.

Project task: [Add Product page](https://www.odoo.com/odoo/my-tasks/5239722)

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+Next, navigate to :menuselection:`Inventory app --> Configfuration --> Locations`. Click :guilabel:`New` to
+configure a new internal location.
+
+In the new location view, enter the :guilabel:`Location Name` and ensure that the :guilabel:`Parent
```
```suggestion
In the new location page, enter the :guilabel:`Location Name` and ensure that the :guilabel:`Parent
```

optional? when i read "view", i think of something website framework related, or maybe a report view

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+Location` field is :guilabel:`WH`. Click the :icon:`fa-cloud-upload` :guilabel:`(Save manually)`
+icon near the top to save.
+
+Rental Products
```
```suggestion
Rental products
```

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+Select the :guilabel:`Product Type` as :guilabel:`Goods` and tick the :guilabel:`Track Inventory`
+checkbox. Next to the :guilabel:`Track Inventory` checkbox, select the :guilabel:`By Unique Serial
+Number` from the drop-down menu. For the :guilabel:`Category` field, select :guilabel:`Goods` from
+the drop-down menu or create a new category by typing in the name and clicking :guilabel:`Create`.
```
more a question rather than feedback for this doc: how does setting a product category be helpful for the business renting out products? It seems optional based on my knowledge, so i would personally omit the entire sentence "For the category... clicking Create". I'm also not certain which product categories are created by default in customer databases, so i'm not sure whether "Goods" is available by default

Felicious replied:
okay!~ let's defer to SME decisions :heart:

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+
+Click the :guilabel:`Rental prices` tab and in the :guilabel:`Pricing` section, click
+:guilabel:`Add a price` to enter a new rental rate. Choose a *pricing period* (:dfn:`the unit of
+duration of the rental`) in the :guilabel:`Period` column, or create a new pricing period by typing
```
good use of the dfn markup! (:

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+   :guilabel:`Ok`, and Odoo recalculates the rental price accordingly.
+
+Once all the information has been entered correctly on the rental order form, click :guilabel:`Send`
+to send the quotation to the customer, or click :guilabel:`Confirm` to confirm the order. A
```
unless im mistaken, i think the actions to confirm a rental order is
1. click send --> email stuff--> confirm order
2. click confirm order 

so how about this smol edit?


```suggestion
to send the quotation to the customer, then click :guilabel:`Confirm` to confirm the order. A
```

meval1006 commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+
+   Navigate to the **Rental** app and select the desired rental order to apply the pricelist to.
+   Click the :guilabel:`Pricelist` field and from the drop-down menu select the newly created
+   pricelist. Click :guilabel:`Update Prices` to apply the pricelist to the total amount.
```
Thanks for the feedback. This is a section I plan to expand based on input from Jose during our second meeting. He recommended several options, so I plan to include them to provide more context.

meval1006 commented:
@Felicious Hi Felicia, I have revised the page according to your comments. The most significant changes are:

- Revising the language to detail that this is for a physical product.
- Reorganized the Create a new physical product section: Added a new note and simplified the Example.

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+===============
+
+To view all products that can be rented in the database, navigate to :menuselection:`Rentals app -->
+Products`. By default, the *Rental* filter appears in the search bar, and the view is Kanban.
```
```suggestion
Products`. By default, the :guilabel:`Rental` filter appears in the search bar, and the view is Kanban.
```

style nitpick: I'd do guilabel here because we've already navigated to this page and we can currently see the Rental filter getting applied

Felicious commented on content/applications/sales/rental/products.rst:
```diff
@@ -0,0 +1,286 @@
+===============
+
+When a customer picks up the product, navigate to the appropriate rental order, click
+:guilabel:`Pickup`. When the *Pickup* page appears, click :guilabel:`Validate`.
```
Something something delivery order (can link to this page: https://www.odoo.com/documentation/19.0/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.html#process-delivery)

meval1006 commented:
@Felicious I have revised the Pickup products and Return products sections.

Felicious commented on content/applications/sales/rental/products/rental-order-bike-rental-1.png:
Are we still using this image?

Felicious reviewed (approved):
Content looks great, @meval1006 !

I think there are two unused images, so after you remove those, this PR is ready to be merged!

@robodoo delegate+

meval1006 commented:
@robodoo r+
